politeiawww/plugin.go

```diff
@@ -62,7 +62,7 @@ func (p *politeiawww) getPluginInventory() ([]plugin, error) {
 			return nil, fmt.Errorf("max retries exceeded")
 		}
 
-		pi, err := p.pluginInventory()
+		pi, err := p.pluginInventory(ctx)
```

Comment:
Wasn't sure how to properly pass a context to `getPluginInventory()`. This function gets called once on startup on `www.go:903`. Is the empty background context fine here?